### PR TITLE
GCWeb theme implementing guide - Create French version

### PR DESCRIPTION
**Pull request by Ricokola**
Related to: WET-268

### CODE REVIEW

**Review by GormFrank — COMMENTED** [review]
@Ricokola Pour éviter le changement de nom de fichier, je viens d'avoir une idée. On pourrait laisser la page anglaise nommée "implementing", puis nommer la française "implementer"?

CC: @duboisp

**Review by duboisp — CHANGES_REQUESTED** [review]
Add a meta-redirect for "implementing.html" and add a comment "Temporary redirect until September 2023"

See this page as an example: https://github.com/wet-boew/themes-dist/blob/f0a0d8ef08cd50840676e96cf0caff29c124a645/demos/index-en.html

**Comment by duboisp** [comment]
Otherwise the redirect page, this PR look good to be merged.

**Comment by Ricokola** [comment]
@duboisp and @GormFrank the redirect page has been created. Please review and merge if everything is good!

File: docs/implementing.html [hunk 1]
```diff
@@ -1,242 +1,15 @@
...
+		/GCWeb/docs/implementing-en.html /GCWeb/docs/implementing-fr.html 
+		This is a temporary redirect and will be removed in September 2023-->
+<title>Quick implementation guide - GCWeb theme
+</title>
+<meta http-equiv="refresh" content="0;https://wet-boew.github.io/GCWeb/docs/implementing.html" />
+</head>
+<body>
+<p>This page has moved to <a href="/GCWeb/docs/implementing-en.html">https://wet-boew.github.io/GCWeb/docs/implementing-en.html</a>.</p>
+</body>
+</html>
```
Comment (duboisp) [inline docs/implementing.html:15]: Need a new line here

**Review by duboisp — CHANGES_REQUESTED** [review]
Other than the new line feed. it looks good to merge.

This is a Content change only

**Review by duboisp — APPROVED** [review]
Requested change was applied.

**Pull request merged**